### PR TITLE
[tests] Add multiple token owner test

### PR DESCRIPTION
**Pull request by Bushstar**
DeFi tokens can have multiple owners using a multisig address for collateral. The test here creates a 1-of-3 multisig and using that address changes the name three times using each of the multisig addresses. This script is also a demonstration of multiple owner tokens.

### CODE REVIEW

File: test/functional/feature_tokens_multisig.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
```
Comment (ShengguangXiao) [inline test/functional/feature_tokens_multisig.py:2]: Can change to 2020

**Comment by ShengguangXiao** [comment]
Can add some failure test cases too?

**Review by ShengguangXiao — CHANGES_REQUESTED** [review]

**Comment by Bushstar** [comment]
@ShengguangXiao I'll add a test for 2-of-3 and we can check that 1 single signature will fail and that 2-of-3 works, also check that a single member of the multisig cannot make changes directly.

**Comment by Bushstar** [comment]
@ShengguangXiao 2-of-3 test case added with above mentioned failure cases.

**Review by ShengguangXiao — APPROVED** [review]

**Comment by monstrobishi** [comment]
This only touches python tests, merging in

**Pull request merged**